signalfx-tracing/signalfx-python-tracing/django/client.py

```diff
@@ -0,0 +1,125 @@
+# IMPORTANT additions to a client file:
+# from signalfx_tracing import auto_instrument, create_tracer, trace
+# from signalfx_tracing.libraries import requests_config
+#
+# requests_config.propagate = True
+# tracer = create_tracer(service_name='<APP_NAME>')
+# auto_instrument(tracer)
+#
+# #Then at the end of all function calls to be traced:
+# tracer.close()
+#
+# These are noted below with additional explanations.
+
+
+import logging
+import requests
+from argparse import ArgumentParser
+from requests.auth import HTTPBasicAuth
+from signalfx_tracing import auto_instrument, create_tracer, trace
+# Important: Needed to propagate traces
+from signalfx_tracing.libraries import requests_config
+
+
+# Create and configure logger
+logging.basicConfig(level='DEBUG',
+                    format='%(asctime)s %(message)s',)
+
+
+# create an instance of the SignalFx tracer
+tracer = create_tracer(service_name='BlogClient')
+
+# Setting requests_config.propagate to 'True'  ensures
+# that the relational information between traces is maintained
```

Comment:
"between spans" (Traces should be discrete and isolated events)

Reply:
Thanks. Corrected!